otelsql/otel.go

```diff
@@ -161,6 +166,13 @@ func WithMeterProvider(meterProvider metric.MeterProvider) Option {
 	}
 }
 
+// WithFormatQueryFunc configures a func to format query string
+func WithFormatQueryFunc(formatQueryFunc func(query string) string) Option {
```

Comment:
Let's rename this to `WithQueryFormatter`. Otherwise, this looks good. :+1:

Reply:
> Let's rename this to `WithQueryFormatter`. Otherwise, this looks good. 👍


done.